src/test/java/com/rocketden/main/service/UserServiceTests.java

```diff
@@ -13,8 +15,9 @@
 import org.mockito.Spy;
 import org.mockito.junit.jupiter.MockitoExtension;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
```

Comment:
nit: personal preference, but I think we should stick with importing each item individually (see [here](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad)). Sometimes the IDE can auto-correct imports into the wildcard tho so usually no big deal if it slips through